src/main/jbake/content/blog/2017/03/15/hawkular-opentracing-otsc.adoc

```diff
@@ -0,0 +1,43 @@
+= Distributed Tracing Workshop and OpenTracing Collaboration
+Pavol Loffay
+2017-03-15
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog
+
+Distributed tracing workshop is organized by https://groups.google.com/forum/#!forum/distributed-tracing[Distributed Tracing Workgroup]
+and takes place approximately once a year. This time it was 18th-19th February in Berlin. It was a pleasure
+to be there with people from various companies and share ideas and a common goal to improve distributed tracing.
+
+== Summary
+
+We have discussed many thing here are my main takeaways:
+
+Key upcoming changes in OpenTracing:
+
+* Formal project organization OTSC (keep reading).
```

Comment:
It's a good practice to spell out terms the first time you use them, like: 

> Formal organization of the OpenTracing Specification Council, or OTSC for short.

Or yet:

> Formal organization of the OpenTracing Specification Council (OTSC).